src/sentry/monitors/tasks.py

```diff
@@ -129,6 +146,30 @@ def try_monitor_tasks_trigger(ts: datetime):
     _dispatch_tasks(ts)
 
 
+@instrumented_task(name="sentry.monitors.tasks.clock_pulse", silo_mode=SiloMode.REGION)
+def clock_pulse(current_datetime=None):
```

Comment:
Shouldn't this run conditionally based on a setting?

Reply:
No, as per fillipos comment https://github.com/getsentry/sentry/issues/53661#issuecomment-1668605281, we decided to just always have it run. Then we don't have a "high" or "low" volume mode. it is just adaptive.

Reply:
If this is running all the time then doesn't it break the case where there's an ingest backlog?

Reply:
No because it is still putting the message into the kafka topic, thus into the queue.

This would be problematic if we turned off producers (relay)